modules/react/checkbox/stories/examples/Indeterminate.tsx

```diff
@@ -45,8 +45,8 @@ export const Indeterminate = () => {
 
     const anyToppingChecked = newToppings.filter(topping => topping.checked).length > 0;
     const anyToppingUnchecked = newToppings.filter(topping => !topping.checked).length > 0;
-    setPizzaChecked(anyToppingChecked);
     setPizzaIndeterminate(anyToppingChecked && anyToppingUnchecked);
+    setPizzaChecked(anyToppingChecked && !anyToppingUnchecked);
```

Comment:
We want the top-level pizza checkbox to only be checked if _all_ toppings are checked, correct? 

```
!anyToppingUnchecked == "not any toppings are unchecked" == "all toppings are checked"
```

```suggestion
    setPizzaChecked(!anyToppingUnchecked);
```

Alternatively, we could just set an `allToppingsChecked` variable for a less confusing way of putting this 😂

Reply:
oof yeah the first part is redundant, I like the idea of making a new var to be more clear on the intentions here